manifests/preflight.yaml

```diff
@@ -4,6 +4,50 @@ metadata:
   name: sentry-enterprise
 spec:
   analyzers:
+    # Fails on preflight page even though registry has been added vendor.replicated.com.
+    - imagePullSecret:
+          checkName: Has Access to index.docker.io
+          registryName: index.docker.io
+          outcomes:
+            - fail:
+                message: Cannot pull from index.docker.io
+            - pass:
+                message: Found credentials to pull from index.docker.io
+    # Fails on preflight page even though Ingress exists.
+    - ingress:
+        namespace: default
+        ingressName: sentry-ingress
+        outcomes:
+          - fail:
+              message: Sentry ingress is not working.
+          - pass:
+              message: Sentry ingress is working.
+    - deploymentStatus:
+        name: sentry-worker
+        namespace: default
+        outcomes:
+          - fail:
+              when: "< 1"
+              message: The Sentry Worker deployment does not have any ready replicas.
+          - warn:
+              when: "= 1"
+              message: The Sentry Worker deployment has only a single ready replica.
+          - pass:
+              message: There are multiple replicas of the Sentry Worker deployment ready.
+    # Is it statefulset or statefulsetStatus?
+    # Does not even show up in preflights page.
+    - statefulsetStatus:
```

Comment:
this looks correct to me (minus `namespace`). I'll look for the root cause of this one.